src/app/portlets/dot-edit-page/content/components/dot-form-selector/dot-form-selector.component.html

```diff
@@ -1,4 +1,4 @@
-<dot-dialog [show]="show" (close)="close.emit($event)" [header]="messages['modes.Add-Form']">
+<dot-dialog [(visible)]="show" (hide)="close.emit($event)" [header]="messages['modes.Add-Form']">
```

Comment:
Don't use `ngIf="show"` in the new `dot-dialog` just use `[(visible)]="show"` and it supports two-way binding, hiding the dialog with close button updates the visible state as false. (just like primeng's one)